### ISSUE TITLE
 Problem about dataset setting and baseline

### ISSUE THREAD

**Issue by 704051419** [Jia-Research-Lab/PFENet#7]
Thanks for sharing your work! I would like to know why the PASCAL training set only contains  about 5900 images. I mean in the setting of CANet or OSLSM, the number of training images is more than that, right?

**Comment by achenzx** [comment]
@704051419 Thanks for your interest. The images provided in the repo of CANet have been pre-processed to binary class-wise labels, while the 5900+ images mentioned in our introduction section are the original multi-class labels of PASCAL VOC and SBD. Therefore the number of class-wise binary labels is greater than that of the multi-class labels. We follow the instruction mentioned in the paper of OSLSM to prepare the training data of PASCAL.

**Comment by 704051419** [comment]
@achenzx  Thanks for your reply? I  am  a little confused about the baseline model. In the Fig.6 in the paper, the baseline structure contains prior mask. But the Table 7 in ablation study has "baseline+Prior". So I am not sure whether the prior mask  is involved in baseline model.

**Comment by achenzx** [comment]
@704051419 Thank you for your suggestion. The caption of Fig.7 should be accordingly modified to the 'baseline model with the prior mask'. The 'Baseline' in Table 7 is the plain structure without the prior mask and FEM.

**Comment by Hardcandies** [comment]
Hi,

> The images provided in the repo of CANet have been pre-processed to binary class-wise labels

What does ‘binary class-wise labels’ mean？

And what you're explaining is the number of 'labels', but I wonder  why the number of 'images' differs from CANet?

I'm sorry I really don't fully understand the relation.

Thank you for your sharing and look forward to your reply!

jachymchen

**Comment by 704051419** [comment]
@Hardcandies there are many images with more than one class, so one image can exist in different class folder. Shortly, there are many repeated images in the folder by CANet. So the  total number is different.

**Comment by Hardcandies** [comment]
I see. Thank you.
